src/components/input/date/range/index.tsx

```diff
@@ -59,17 +101,20 @@ export class SmoothlyInputDateRange implements Clearable, Input {
 	render() {
 		return (
 			<Host tabindex={0}>
-				<section onClick={() => (this.open = !this.open)}>
+				<section onClick={() => !this.readonly && (this.open = !this.open)}>
 					<smoothly-input
 						type="text" // TODO: date-range tidily thing
 						name="dateRangeInput"
-						value={`${this.start ? this.start : "from "} - ${this.end ? this.end : " to"}`}
+						readonly={this.readonly}
+						value={this.start && this.end ? `${this.start} - ${this.end}` : undefined}
```

Comment:
Use en dash, "–", instead of hyphen.

Reply:
Added a so called Em Dash on both the placeholder and on when the value is set.